diff -p -up gnomebaker-0.6.4/src/gnomebaker.c.strfmt gnomebaker-0.6.4/src/gnomebaker.c
--- gnomebaker-0.6.4/src/gnomebaker.c.strfmt	2009-12-12 18:18:07.000000000 +0100
+++ gnomebaker-0.6.4/src/gnomebaker.c	2009-12-12 18:31:13.000000000 +0100
@@ -225,7 +225,7 @@ gnomebaker_show_msg_dlg(GtkWindow *paren
 
 	GtkWidget *dialog = gtk_message_dialog_new(
             parent == NULL ? GTK_WINDOW(glade_xml_get_widget(xml, widget_gnomebaker)) : parent,
-		    GTK_DIALOG_DESTROY_WITH_PARENT, type, buttons, message);
+		    GTK_DIALOG_DESTROY_WITH_PARENT, type, buttons, "%s", message);
 
 	/*if(additional != GTK_BUTTONS_NONE)
 		gtk_dialog_add_button(GTK_DIALOG(dialog), GTK_STOCK_CANCEL, GTK_RESPONSE_CANCEL);*/
